tutorials/FITS-images/FITS-images.ipynb

```diff
@@ -392,43 +699,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "Diir_0hlgEIy"
+   },
    "source": [
     "## Exercises"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "wpPk3Zv6gEIy"
+   },
    "source": [
-    "Determine the mean, median, and standard deviation of a part of the stacked M13 image where there is *not* light from M13.  Use those statistics with a sum over the part of the image that includes M13 to estimate the total light in this image from M13."
+    "Determine the mean, median, and standard deviation of the stacked M13 image data.  Add corresponding keywords 'MEAN', 'MEDIAN', and 'STANDARD DEVIATION' to the primary HDU in the outfile you just writed to, and store the statistics you calculated in the keywords created in the header. See the [Editing a FITS Header tutorial](https://learn.astropy.org/tutorials/FITS-header.html) to learn how to add the keywords. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "r3W9AATEgEIy"
+   },
    "outputs": [],
    "source": []
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "IZmcn4vfgEIy"
+   },
    "source": [
     "Show the image of the Horsehead Nebula, but in units of *surface brightness* (magnitudes per square arcsecond).\n",
-    "(Hint: the *physical* size of the image is 15x15 arcminutes.)"
+    "(Hint: the *physical* size of the image is 15x15 arcminutes.) For the Horsehead Nebula image used in this tutorial, assume that the magnitude is just the value stored in each pixel."
```

Comment:
Thanks for this -- I have wondered for years how to do this without some kind of assumption like the one you are giving here!